src/MixedModels.jl

```diff
@@ -208,6 +210,11 @@ include("profile/profile.jl")
 include("nlopt.jl")
 include("prima.jl")
 
+
+# aliases with non-unicode function names
+settheta! = setθ!
+profilesigma = profileσ
```

Comment:
```suggestion
const settheta! = setθ!
const profilesigma = profileσ
```

Setting these `const` helps with all sorts of little performance things.